File: Starters/Starters 47/CHESSTIME/Solution.cpp
#include <iostream>
using namespace std;

int main() {
	int t,i,n,chess;
	cin>>t;
	for(i=0;i<t;i++){
	    cin>>n;
	    chess = (n*60)/20;
	    cout<<chess<<"\n";
	}
	return 0;
}

File: Starters/Starters 47/CHESSTIME/Solution.go
package main
import "fmt"

func main() {
	var t,n,chess int
	fmt.Scanln(&t)
  for i:=0;i<t;i++ {
	    fmt.Scanln(&n)
	    chess = (n*60)/20;
	    fmt.Println(chess);
	}
}
